distribution/openrct2.d.ts

```diff
@@ -3213,6 +3238,7 @@ declare global {
         /**
          * The research category this item belongs in.
          * E.g. gentle rides, thrill rides, shops etc.
+         * Note: This field is ignored by OpenRCT2, the category will be determined by the ride type.
```

Comment:
```suggestion
         * Note: Any updates to this field are ignored by OpenRCT2, the category will be derived from the ride type.
```
As it's not ignored when getting/reading the property.